scripts/abi_check.py

```diff
@@ -208,6 +230,92 @@ def _get_abi_dumps_from_shared_libraries(self, version):
             self.log.debug(abi_dump_output.decode("utf-8"))
             version.abi_dumps[mbed_module] = output_path
 
+    @staticmethod
+    def _normalize_storage_test_case_data(line):
+        """Eliminate cosmetic or irrelevant details in storage format test cases."""
+        line = re.sub(r'\s+', r'', line)
```

Comment:
But it would be different. The call to `re.sub` here also removes internal whitespace (e.g. `myfunc:PSA_FOO(PSA_BAR)` vs `myfunc:PSA_FOO( PSA_BAR )`).